### PR TITLE
Adding Numeric Keypad //e converter

### PR DESCRIPTION
**Pull request by newbold**
This is a new keyboard conversion for the Numeric Keypad //e, Apple's first non-integrated keyboard which happened to be a numpad for the Apple //e.

## Description

Added keyboards/converter/numeric_keypad_IIe with everything needed to compile working QMK firmware on an ATmega32U4 microcontroller.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/converter/numeric_keypad_IIe/config.h [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Apple Inc.
+#define PRODUCT         Numeric Keypad //e
```
Comment (noroadsleft) [inline keyboards/converter/numeric_keypad_IIe/config.h]: As someone whose first experience with mechanical keyboards was an AEK II (before I even knew what a mechanical keyboard was), I totally get the reason for using `//e` here, but I think this should be changed to:

```suggestion
#define PRODUCT         Numeric Keypad IIe
```

File: keyboards/converter/numeric_keypad_IIe/config.h [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+#define DESCRIPTION     Numeric Keypad //e, A2M2003
```
Comment (noroadsleft) [inline keyboards/converter/numeric_keypad_IIe/config.h]: I did say on Discord that commas aren't allowed in the USB Device Descriptor, but looking this over, I made a realization so obvious that I'm actually kind of embarrassed to admit to it now.

Wrap the value in quotation marks and you can use commas in the USB Device Descriptor.

```suggestion
#define DESCRIPTION     "Numeric Keypad IIe, A2M2003"
```

Made this change locally and it compiles with no errors.

**Comment by newbold** [comment]
@noroadsleft Thanks, I committed both of those changes. Really appreciate the feedback!

File: keyboards/converter/numeric_keypad_IIe/info.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Numeric Keypad IIe", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "width": 7.5, 
+    "height": 4, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"label":"Esc", "x":0, "y":0}, {"label":"7", "x":2, "y":0}, {"label":"8", "x":3, "y":0}, {"label":"9", "x":4, "y":0}, {"label":"(", "x":5.5, "y":0}, {"label":")", "x":6.5, "y":0}, {"label":"\u2190", "x":0, "y":1}, {"label":"4", "x":2, "y":1}, {"label":"5", "x":3, "y":1}, {"label":"5", "x":4, "y":1}, {"label":"\u2212", "x":5.5, "y":1}, {"label":"\u00f7", "x":6.5, "y":1}, {"label":"\u2192", "x":0, "y":2}, {"label":"1", "x":2, "y":2}, {"label":"2", "x":3, "y":2}, {"label":"3", "x":4, "y":2}, {"label":"+", "x":5.5, "y":2}, {"label":"\u00d7", "x":6.5, "y":2}, {"label":"Space", "x":0, "y":3}, {"label":"0", "x":1.5, "y":3, "w":1.5}, {"label":",", "x":3, "y":3}, {"label":".", "x":4, "y":3}, {"label":"Ret", "x":5.5, "y":3}, {"label":"Print", "x":6.5, "y":3}]
```
Comment (noroadsleft) [inline keyboards/converter/numeric_keypad_IIe/info.json]: You have two `5` keys and no `6`.

```suggestion
            "layout": [{"label":"Esc", "x":0, "y":0}, {"label":"7", "x":2, "y":0}, {"label":"8", "x":3, "y":0}, {"label":"9", "x":4, "y":0}, {"label":"(", "x":5.5, "y":0}, {"label":")", "x":6.5, "y":0}, {"label":"\u2190", "x":0, "y":1}, {"label":"4", "x":2, "y":1}, {"label":"5", "x":3, "y":1}, {"label":"6", "x":4, "y":1}, {"label":"\u2212", "x":5.5, "y":1}, {"label":"\u00f7", "x":6.5, "y":1}, {"label":"\u2192", "x":0, "y":2}, {"label":"1", "x":2, "y":2}, {"label":"2", "x":3, "y":2}, {"label":"3", "x":4, "y":2}, {"label":"+", "x":5.5, "y":2}, {"label":"\u00d7", "x":6.5, "y":2}, {"label":"Space", "x":0, "y":3}, {"label":"0", "x":1.5, "y":3, "w":1.5}, {"label":",", "x":3, "y":3}, {"label":".", "x":4, "y":3}, {"label":"Ret", "x":5.5, "y":3}, {"label":"Print", "x":6.5, "y":3}]
```

Comment (newbold) [reply keyboards/converter/numeric_keypad_IIe/info.json]: Fixed, thanks!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**